jecs.d.ts

```diff
@@ -40,13 +40,39 @@ type FlattenTuple<T extends unknown[]> = T extends [infer U] ? U : LuaTuple<T>;
 type Nullable<T extends unknown[]> = { [K in keyof T]: T[K] | undefined };
 type InferComponents<A extends Id[]> = { [K in keyof A]: InferComponent<A[K]> };
 
+type i53 = number
+type i24 = number
+type Ty =  i53[];
+type ArchetypeId = number
+type Column = unknown[];
```

Comment:
I think there can be some generics work done here to actually try to infer the type here. I don't like how this looks right now, but we can probably take a look at this in the future.